assets/templates/homepage/latest-releases.tmpl

```diff
@@ -0,0 +1,26 @@
+{{ $releases := len .Data.ReleaseCalendar.Releases }}
```

Comment:
This breaks as we are setting the releases to an integer but then wanting to iterate over it.

```suggestion
{{ $releases := .Data.ReleaseCalendar.Releases }}
```

Reply:
Good spot. C&P error I think